db/db.go

```diff
@@ -238,6 +251,18 @@ func (db *Connection) GetRecordsOfType(deviceID string, limit int, eventType int
 	return deviceInfo, nil
 }
 
+// GetRecords returns a list of records for a given device
+func (db *Connection) GetBlackList() (blacklist []BlackDevice, err error) {
```

Comment:
Here this function is called `GetBlackList()` but in the retry functions and interface it's called `GetBlacklist()`.  The L needs to be either lowercase or capitalized everywhere.